cfme/tests/automate/test_git_import.py

```diff
@@ -352,3 +352,28 @@ def _cleanup():
         domain.delete_if_exists()
 
     assert automate_domain.exists
+
+
+@pytest.mark.tier(1)
+def test_automate_git_import_case_insensitive(request, appliance):
+    """
+    bin/rake evm:automate:import PREVIEW=false
+    GIT_URL=https://github.com/mkanoor/SimpleDomain REF=test2branch
+    This should not cause an error (the actual name of the branch is
+    Test2Branch).
+
+    Polarion:
+        assignee: ghubale
+        casecomponent: Automate
+        caseimportance: medium
+        initialEstimate: 1/8h
+        tags: automate
+        startsin: 5.7
+    """
+    appliance.ssh_client.run_rake_command(
+        "evm:automate:import PREVIEW=false GIT_URL=https://github.com/ganeshhubale/SimpleDomain "
```

Comment:
@ganeshhubale, your test looks good but I doubt @mshriver will approve using your own repo. Could you use one of our official repos for that  instead ?

Reply:
@izapolsk Yes I agree with you.
So I am trying this test case with repository from ```RedHatQE``` group.
Repository URL: https://github.com/RedHatQE/ManageIQ-automate-git